test/Dockerfile.test-project

```diff
@@ -0,0 +1,7 @@
+FROM InsertBaseImageHere
```

Comment:
Can you add a comment that this is overridden in CI? It's not obvious without ctrl+f

Reply:
249f1b1e0ee5ab2d85de8f0fb774cc8d58264067